graphing/html/index.html

```diff
@@ -3,60 +3,135 @@
+            //             var yAxe = chart.scales[chart.config.options.scales.yAxes[0].id];
+            //
+            //
+            //             if(yAxe.min != 0) return;
+            //
+            //             ctxPlugin.strokeStyle = "grey";
+            //             ctxPlugin.beginPath();
+            //             lineAt = (lineAt - yAxe.min) * (100 / yAxe.max);
+            //             lineAt = (100 - lineAt) / 100 * (yAxe.height) + yAxe.top;
+            //             ctxPlugin.moveTo(xAxe.left, lineAt);
+            //             ctxPlugin.lineTo(xAxe.right, lineAt);
+            //             ctxPlugin.stroke();
+            //         }
+            //     }
+            // });
             const myChart = new Chart(ctx, {
-                type: 'bar',
+                type: 'line',
                 data: {
-                    labels: ['Red', 'Blue', 'Yellow', 'Green', 'Purple', 'Orange'],
+                    labels: ['1', '2', '3', '4', '5', '6', '7','8', '9', '10', '11','12'],
                     datasets: [{
-                        label: '# of Votes',
-                        data: [12, 19, 3, 5, 2, 3],
+                        label: false,
+                        data: [0.6, 0.75, 0.4, 0.65, 0.9, 1.1, 0.78, 1.5, 1.05, 0.4, 0.8, 0.6],
+                        lineTension: 0,
+                        borderColor: 'rgb(128,23,25)',
+                        fill: false,
                         backgroundColor: [
-                            'rgba(255, 99, 132, 0.2)',
-                            'rgba(54, 162, 235, 0.2)',
-                            'rgba(255, 206, 86, 0.2)',
-                            'rgba(75, 192, 192, 0.2)',
-                            'rgba(153, 102, 255, 0.2)',
-                            'rgba(255, 159, 64, 0.2)'
-                        ],
-                        borderColor: [
-                            'rgba(255, 99, 132, 1)',
-                            'rgba(54, 162, 235, 1)',
-                            'rgba(255, 206, 86, 1)',
-                            'rgba(75, 192, 192, 1)',
-                            'rgba(153, 102, 255, 1)',
-                            'rgba(255, 159, 64, 1)'
+                            //'rgba(255, 99, 132, 0.2)',
+                            // 'rgba(54, 162, 235, 0.2)',
+                            //'rgba(255, 206, 86, 0.2)',
+                             'rgba(75, 192, 192, 0.2)',
+                            // 'rgba(153, 102, 255, 0.2)',
+                            // 'rgba(255, 159, 64, 0.2)'
                         ],
-                        borderWidth: 1
+                        borderWidth: 5
                     }]
                 },
                 options: {
+                    legend: {
+                        display: false,
+                    },
+                    title: {
+                        display: true,
+                        text: 'Median time to first view after first sustained eCART elevation, hrs'
```

Comment:
This too is dynamic so will need configuration for the graph. To do that, create another API route so that you have both a route for the data and one for which graph to generate.

One route will be `/data` that returns an array of dataPoints which are a `value` and `label` pair.